### PR TITLE
Bump Laravel 11 and make use of common/markdown

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
@samsonasik While we're at it, could you check this one and eploy it too? :pray:

**Comment by samsonasik** [comment]
This looks good, let's try 👍

File: .env.local [hunk 1]
```diff
@@ -1,4 +1,8 @@
 APP_ENV=dev
 APP_DEBUG=true
-APP_KEY=random_key
+APP_KEY=base64:NKiscQs2YqBLEa8qt7cbRKNCUef6LvD7PR5PQW7wDTk=
 APP_URL=http://localhost
+
+# required for Laravel defautlst to work
+DB_CONNECTION=sqlite
+DB_DATABASE=database/database.sqlite
```
Comment (samsonasik) [inline .env.local:8]: Running `php82 artisan cache:clear` seems got error:

```
➜  tomasvotruba.com git:(tv-laravel-11) php82 artisan cache:clear

In Connection.php line 806:
                                                                                                         
  SQLSTATE[HY000]: General error: 1 no such table: cache (Connection: sqlite, SQL: delete from "cache")  
                                                                                                         

In Connection.php line 585:
                                                          
  SQLSTATE[HY000]: General error: 1 no such table: cache  
```

The website seems ok, but it seems this issue for clearing cache process. I will look more.

Comment (samsonasik) [reply .env.local:8]: I see, cache migration table seems not exists, I will add it.

Comment (samsonasik) [reply .env.local:8]: https://github.com/TomasVotruba/tomasvotruba.com/pull/1440

**Pull request merged**